pkg/aws/s3blobstore.go

```diff
@@ -154,6 +157,11 @@ func (s *S3BlobStore) Get(ctx context.Context, digest multihash.Multihash, opts
 		Range:  rangeParam,
 	})
 	if err != nil {
+		var noSuchKeyError *types.NoSuchKey
+		// wrap in error recognizable as a not found error for Store interface consumers
+		if errors.As(err, &noSuchKeyError) {
+			return nil, store.NewErrNotFound(err)
```

Comment:
Sorry needs more change - this *i think*:

```suggestion
			return nil, store.ErrNotFound
```